sources/ClangSharp/Generated.Custom.cs

```diff
@@ -1,325 +0,0 @@
-namespace ClangSharp
```

Comment:
I was able to get rid of all method/type exclusions and just generate the bindings directly instead.